src/mon/OSDMonitor.cc

```diff
@@ -1128,30 +1129,46 @@ void OSDMonitor::encode_pending(MonitorDBStore::TransactionRef t)
 
   // tell me about it
   for (auto i = pending_inc.new_state.begin();
-       i != pending_inc.new_state.end();
-       ++i) {
+    i != pending_inc.new_state.end(); ++i) {
```

Comment:
@shinobu-x I don't want to be too annoying about formatting issues, but some of your changes just ignore the spacing and alignment of lines with their parents. In this case, the line was previous aligned but no longer is. There are a few other instances below.